pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -87,8 +117,15 @@ func (ds *dockerService) determinePodIPBySandboxID(sandboxID string) string {
 		// Todo: Add a kernel version check for more validation
 
 		if networkMode := os.Getenv("CONTAINER_NETWORK"); networkMode == "" {
-			// Do not return any IP, so that we would continue and get the IP of the Sandbox
-			ds.getIP(sandboxID, r)
+			if r.HostConfig.Isolation == hypervIsolation {
+				// Return the first non-sandbox container IP as POD IP for Hyper-V container.
```

Comment:
Can you explain this to me?

Reply:
As noticed [here](https://github.com/kubernetes/kubernetes/pull/58751#issue-291226175), HyperV only support one container per Pod yet. And container will receive a different IP from sandbox. So we need to get the real container's IP.

Reply:
Could you include information that in the comment?

Reply:
added